elliottlib/bzutil.py

```diff
@@ -1058,6 +1059,27 @@ def get_highest_security_impact(bugs):
     return 'Low'
 
 
+def compare_impact(bug_a, bug_b):
+    """
+    Check if CVE priority of bug_a if lower than bug_b
+    first compare severity level, if severity level is the same
+    then compare CVE number
+    """
+    buga_l = constants.security_impact_map[get_highest_security_impact([bug_a])]
+    bugb_l = constants.security_impact_map[get_highest_security_impact([bug_b])]
+    if buga_l == bugb_l:
+        if bug_a.alias[0].split("-")[1] == bug_b.alias[0].split("-")[1]:
+            return 1 if int(bug_a.alias[0].split("-")[2]) < int(bug_b.alias[0].split("-")[2]) else -1
+        else:
+            return 1 if int(bug_a.alias[0].split("-")[1]) < int(bug_b.alias[0].split("-")[1]) else -1
+    else:
+        return 1 if buga_l > bugb_l else -1
+
```

Comment:
This can be simplified further.

Python supports comparing two lists or tuples:
```python
(2021, 1) < (2021, 2)
(2022, 1) > (2021, 2)
```

So something like the following should work:

```suggestion
def cve_sort_key(bug):
    impact = constants.security_impact_map[get_highest_security_impact([bug])]
    year, num = bug.alias[0].split("-")[1:]
    return impact, year, num
```

Then sort it with:
return sorted(bugs, key=cve_sort_key, reverse=True)

Reply:
thanks for you suggestion, I updated